lib/widgets/components/receipt/receipt_overlay.dart

```diff
@@ -13,30 +13,31 @@ class ReceiptOverlay {
     Navigator.of(_context).pop();
   }
 
-  void show(Receipt receipt, {String? optionalText}) {
+  void show({
+    required Receipt receipt,
+    required bool isTestEnvironment,
+    String? optionalText,
+  }) {
     showDialog(
       context: _context,
       barrierColor: AppColor.scrim,
       builder: (_) {
-        return Padding(
-          padding: const EdgeInsets.all(48),
-          child: Center(
-            child: Column(
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: [
-                ReceiptCard(
-                  productName: receipt.productName,
-                  time: receipt.timeUsed,
-                  isPurchase:
-                      receipt.transactionType == TransactionType.purchase,
-                  isInOverlay: true,
-                ),
-                Text(
-                  Strings.receiptTapAnywhereToDismiss,
-                  style: AppTextStyle.explainerBright,
-                )
-              ],
-            ),
+        return Center(
```

Comment:
Seems like the padding that should wrap this widget is removed.

Reply:
![image](https://user-images.githubusercontent.com/21163286/158472972-f3db4742-dd0a-4fa9-b65a-ca51045e95e5.png)